CRM/Contribute/Form/CancelSubscription.php

```diff
@@ -136,7 +132,7 @@ public function setEntityFields() {
       'cancel_reason' => ['name' => 'cancel_reason'],
     ];
     $this->entityFields['send_cancel_request'] = [
-      'title' => ts('Send cancellation request to %1 ?', [1 => $this->_paymentProcessorObj->_processorName]),
+      'title' => ts('Send cancellation request?'),
```

Comment:
This checkbox is already confusing to the user - if we remove the reference to the payment processor it feels even more unclear about what it actually does?

Reply:
No - the reference is resolved later on rather than removed - the information is not reliably available at this point

Reply:
ok

Reply:
thanks @mattwire - will you have time to review this? (It's a recent regression so I'd like to get it out)